src/components/organisms/PaymentModal/PaymentModal.tsx

```diff
@@ -8,7 +8,6 @@ import { hasUserBoughtTicketForEvent } from "utils/hasUserBoughtTicket";
 import { isUserAMember } from "utils/isUserAMember";
 import { Purchase } from "types/Purchase";
 import "./PaymentModal.scss";
-import PaymentForm from "./PaymentForm";
 import PaymentConfirmation from "./PaymentConfirmation";
```

Comment:
Presumably none of these will be relevant with the payment stuff removed?

Reply:
fixed in 86ea8ad01c98485b02bc0029048517d83d6b9f3d